src/main/java/com/minecolonies/coremod/colony/requestsystem/data/StandardRequestSystemBuildingDataStore.java

```diff
@@ -0,0 +1,212 @@
+                final NBTTagCompound entryCompound = new NBTTagCompound();
+
+                entryCompound.setTag(TAG_TOKEN, controller.serialize(iToken));
+                entryCompound.setTag(TAG_VALUE, controller.serialize(standardRequestSystemBuildingDataStore.citizenByOpenRequest.get(iToken)));
+
+                return entryCompound;
+            }).collect(NBTUtils.toNBTTagList()));
+
+            return compound;
+        }
+
+        @NotNull
+        @Override
+        public StandardRequestSystemBuildingDataStore deserialize(@NotNull final IFactoryController controller, @NotNull final NBTTagCompound nbt) throws Throwable
+        {
+            final IToken<?> token = controller.deserialize(nbt.getCompoundTag(TAG_TOKEN));
+            final Map<TypeToken<?>, Collection<IToken<?>>> openRequestsByRequestableType = NBTUtils
+                                                                                             .streamCompound(nbt.getTagList(TAG_OPEN_REQUESTS_BY_TYPE, Constants.NBT.TAG_COMPOUND)).map(nbtTagCompound -> {
+                  final TypeToken<?> key = controller.deserialize(nbtTagCompound.getCompoundTag(TAG_TOKEN));
+                  final Collection<IToken<?>> values = NBTUtils.streamCompound(nbtTagCompound.getTagList(TAG_LIST, Constants.NBT.TAG_COMPOUND))
+                                                         .map(elementCompound -> (IToken<?>) controller.deserialize(elementCompound))
+                                                         .collect(Collectors.toList());
+
+                  return new Tuple<>(key, values);
+              }).collect(Collectors.toMap(Tuple::getFirst, Tuple::getSecond));
+            final Map<Integer, Collection<IToken<?>>> openRequestsByCitizen = NBTUtils
+                                                                                             .streamCompound(nbt.getTagList(TAG_OPEN_REQUESTS_BY_CITIZEN, Constants.NBT.TAG_COMPOUND)).map(nbtTagCompound -> {
+                  final Integer key = controller.deserialize(nbtTagCompound.getCompoundTag(TAG_TOKEN));
+                  final Collection<IToken<?>> values = NBTUtils.streamCompound(nbtTagCompound.getTagList(TAG_LIST, Constants.NBT.TAG_COMPOUND))
+                                                         .map(elementCompound -> (IToken<?>) controller.deserialize(elementCompound))
+                                                         .collect(Collectors.toList());
+
+                  return new Tuple<>(key, values);
+              }).collect(Collectors.toMap(Tuple::getFirst, Tuple::getSecond));
+            final Map<Integer, Collection<IToken<?>>> completedRequestsByCitizen = NBTUtils
+                                                                                .streamCompound(nbt.getTagList(TAG_COMPLETED_REQUESTS_BY_CITIZEN, Constants.NBT.TAG_COMPOUND)).map(nbtTagCompound -> {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this 198 characters long line (which is greater than 180 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS00103)